lib/explorer/data_frame.ex

```diff
@@ -1293,6 +1293,17 @@ defmodule Explorer.DataFrame do
 
   * `id_cols` - A set of columns that uniquely identifies each observation. Defaults to all columns in data except for the columns specified in `names_from` and `values_from`. Typically used when you have redundant variables, i.e. variables whose values are perfectly correlated with existing variables. May accept a filter callback or list of column names.
   * `names_prefix` - String added to the start of every variable name. This is particularly useful if `names_from` is a numeric vector and you want to create syntactic variable names.
+
+  ## Examples
+
+      iex> df = Explorer.DataFrame.from_map(%{id: [1, 1], variable: ["a", "b"], value: [1, 2]})
```

Comment:
Somewhat unrelated: another reason to call this function `from_series` is that we could receive any enumerable as a series. This could be beneficial because keywords preserve key ordering but maps do not.

Reply:
Oh good point. No reason not to take a keyword list here. I'll raise an issue and make the change.